packaging/systemd/configure-ovs.sh

```diff
@@ -2,7 +2,7 @@
 
 # source: https://github.com/openshift/machine-config-operator/blob/master/templates/common/_base/files/configure-ovs-network.yaml
 
-set -eux
+set -eu
```

Comment:
```suggestion
set -eux
```

Please leave the -x, I expect we will have to debug this in some cases and better to have full info

Reply:
Reverted the change as requested